frontend/src/components/common/edit-category-modal/styles.module.scss

```diff
@@ -0,0 +1,4 @@
+.buttonWrapper {
+  display: flex;
```

Comment:
You shouldn't add display flex for button wrapper, button should fill all it's width, you should align wrapper instead

Reply:
@Liza-Veis  to align wrapper, its parent should have display flex, shouldn't it? And when the parent of wrapper has display flex, the button takes a minimum width, so there's no more need for using align